test/bats.bats

```diff
@@ -872,12 +871,12 @@ EOF
   ! wait $SUBPROCESS_PID
 
   run cat "$TEMPFILE"
-  echo "$output"
+  
 
-  [[ "${lines[1]}" == "not ok 1 test" ]]
-  [[ "${lines[2]}" == "# (in test file ${RELATIVE_FIXTURE_ROOT}/hang_in_run.bats, line 7)" ]]
-  [[ "${lines[3]}" == "#   \`run sleep 10' failed with status 130" ]]
-  [[ "${lines[4]}" == "# Received SIGINT, aborting ..." ]]
+  [[ "${lines[*]}" =~ "not ok 1 test" ]]
```

Comment:
I think the test was more stringent before. This might allow for reordered output now.